api.md

```diff
@@ -295,7 +284,6 @@ An error will be given if:
 | `port`               | Port to use on the remote data node. The default is the PostgreSQL port used by the access node on which the function is executed. |
 | `if_not_exists`      | Do not fail if the data node already exists. The default is `FALSE`. |
 | `bootstrap`          | Bootstrap the remote data node. The default is `TRUE`. |
-| `bootstrap_database` | Database to use when bootstrapping as described above. The default is `'postgres'`. |
```

Comment:
Is this optional at all?  Or it /only/ supports `postgres` or `template1`?  (If both postgres and template1 supported, which one?

Reply:
Logic here is equal to the way how PostgreSQL itself handles it. I think we support only those two choices for now. It will try to connect to the `postgres` database first and if it fails then try `template1`: https://github.com/timescale/timescaledb/blob/master/tsl/src/data_node.c#L552